diff --git a/components/0030-user-setup b/components/0030-user-setup
index 124038f..e19fcdc 100755
--- a/components/0030-user-setup
+++ b/components/0030-user-setup
@@ -67,7 +67,7 @@ Config ()
 	# Default password is: live
 	# passwords can be generated with 'echo "live" | mkpasswd -s',
 	# a blank password is 'U6aMy0wojraho'.
-	_PASSWORD="8Ab05sVQ4LLps"
+	_PASSWORD="U6aMy0wojraho"
 
 cat > /tmp/live-config.cfg << EOF
 user-setup passwd/make-user boolean true
